### DISCUSSION TITLE
Show Coral TPU Temperature in the System Tab

### DISCUSSION THREAD

**Discussion by yodaphone** [blakeblackshear/frigate#10657] [Ideas]
Came across this shell script that shows the temp of the Coral TPUs. thought it might be useful to add this to the "System" tab along with the CPU % & MEMORY %

https://gist.github.com/dataslayermedia/fb20aae9dea64bca286b5e6d189027b8

**Comment by NickM-27** [comment]
This is implemented in 0.14

**Reply by dz4news** [reply]
Is there a way to change the temp readout to Fahrenheit or is it defaulted to the device?

**Reply by NickM-27** [reply]
In general the standard unit of temperature for computers is Celsius.
